docs/static/releasenotes.asciidoc

```diff
@@ -3,12 +3,22 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-7-0-0,Logstash 7.0.0>>
 * <<logstash-7-0-0-rc2,Logstash 7.0.0-rc2>>
 * <<logstash-7-0-0-rc1,Logstash 7.0.0-rc1>>
 * <<logstash-7-0-0-beta1,Logstash 7.0.0-beta1>>
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-0-0]]
+=== Logstash 7.0.0 Release Notes
+
+coming[7.0.0]
+
+The list combines release notes from the 7.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
```

Comment:
I added a placeholder and "coming soon" label for 7.0.0.  I left the 7.0.0 section empty for now. No point copying all of the other content over and then having to make any corrections in two places.